### PR TITLE
Update to scoped packages

### PR DESCRIPTION
**Pull request by agubler**
**Type:** bug

The following has been addressed in the PR:

* [ ] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [x] Unit or Functional tests are included in the PR

**Description:**

update references to scoped packages.

### CODE REVIEW

File: src/plugins/CoreLoadPlugin.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export default class DojoLoadPlugin {
 		const bundleLoader = /bundle.*\!/;
 		const issuers: string[] = [];
 
-		compiler.apply(new NormalModuleReplacementPlugin(/dojo-core\/load\.js/, resolveMid('dojo-core/load/webpack')));
+		compiler.apply(new NormalModuleReplacementPlugin(/@dojo/core\/load\.js/, resolveMid('@dojo/core/load/webpack')));
```
Comment (mwistrand) [inline src/plugins/CoreLoadPlugin.ts]: This needs an escape after `dojo`: `/@dojo/core` => `/@dojo\/core`.

Comment (agubler) [inline src/plugins/CoreLoadPlugin.ts]: Course, thanks.

Comment (agubler) [inline src/plugins/CoreLoadPlugin.ts]: @mwistrand does the `@` need escaping too?

Comment (mwistrand) [inline src/plugins/CoreLoadPlugin.ts]: `@` is not a control/special character, so no, it should not need to be escaped.

Comment (agubler) [inline src/plugins/CoreLoadPlugin.ts]: i realised that after I said it, but thank you again.

**Comment by mwistrand** [comment]
This issue (and code from this PR) merged with #45.

**Pull request closed without merging**